### ISSUE TITLE
Updating RoboSharp 1.2.8 Throws System.NotSupportedException

### ISSUE THREAD

**Issue by wspait** [tjscience/RoboSharp#163]
Hello,

I recently updated RoboSharp to 1.2.8 from 1,2,5 and am noticing that an exception is being thrown from RoboShrap on application startup:

System.NotSupportedException: 'No data is available for encoding 437. For information on defining a custom encoding, see the documentation for the Encoding.RegisterProvider method.'

This issue seems to happen on both 1.2.7 and 1.2.8. Version 1.2.5 the application will startup. Has anything changed between these versions? We are currently running our app on .NET 6.

**Comment by r-o-b-o-t-o** [comment]
I was experiencing the same issue with RoboSharp 1.2.8, I solved it by adding this line at the start of my application:
```csharp
System.Text.Encoding.RegisterProvider(System.Text.CodePagesEncodingProvider.Instance);
```


*Edit:*
*Looks like this is fixed by #161 : https://github.com/tjscience/RoboSharp/pull/161/files#diff-d5c7a3ee9f1069a0daf83906fa9a13de675d2319e2175eda21fb06486a404979R39*

**Comment by PCAssistSoftware** [comment]
The fix for this error has been implemented in latest commit
